packages/generator/tests/acceptance/test_client_generator.py

```diff
@@ -9,14 +9,18 @@
 
 from tests.conftest import CliRunnerFunction
 from tests.utilities.discovery_service_process import DiscoveryServiceProcess
-from tests.utilities.measurements import non_streaming_data_measurement, streaming_data_measurement
+from tests.utilities.measurements import (
+    non_streaming_data_measurement,
+    streaming_data_measurement,
+    void_measurement,
+)
 
 
-def test___command_line_args___create_client___render_without_error(
+def test___command_line_args_for_non_streaming_measurement___create_client___render_without_error(
```

Comment:
If every precondition says "command line args" then you can remove it from all of them. Just describe what's different.

- non_streaming_measurement
- void_measurement
- all_registered_measurements
- interactive_mode
- etc.

Reply:
I have updated the names of the tests by removing `command_line_args`.